theme-data/dark/dark-common.json

```diff
@@ -243,32 +253,56 @@
         }
       }
     },
-    "illustrations": {
-      "empty": {
-        "primary": ["@color-lime-40", "@color-cobalt-50"],
-        "secondary": ["@color-cobalt-50", "@color-violet-60"]
+    "indicator": {
+      "unstable": "@color-orange-50",
+      "attention": "@color-red-50",
+      "stable": "@color-green-50",
+      "locked": "@color-gray-50",
+      "caution": "@color-yellow-50",
+      "secure": "@color-blue-50"
+    },
+    "whiteboard": {
+      "sticky": {
+        "note": {
+          "yellow": "@color-yellow-30",
+          "red": "@color-red-40",
+          "purple": "@color-purple-40",
+          "cyan": "@color-cyan-40",
+          "mint": "@color-mint-40"
+        }
       },
-      "onboarding": {
-        "primary": ["@color-yellow-30", "@color-lime-40"],
-        "secondary": ["@color-red-50", "@color-violet-60"],
-        "tertiary": ["@color-yellow-30", "@color-orange-50"],
-        "quartenary": ["@color-lime-40", "@color-cobalt-50"],
-        "quinary": ["@color-cobalt-50", "@color-violet-60"]
-      },
-      "success": ["@color-green-40", "@color-lime-40"],
-      "error": ["@color-pink-50", "@color-red-60"],
-      "default": "@color-white-alpha-100"
+      "stickyNote": {
+        "yellow": "@color-yellow-30",
+        "red": "@color-red-40",
+        "purple": "@color-purple-40",
+        "cyan": "@color-cyan-40",
+        "mint": "@color-mint-40"
+      }
     },
-    "globaltint": {
-      "normal": "@color-blue-50",
-      "selected": "@color-blue-60"
+    "ios": {
+      "global": {
+        "tint": {
+          "normal": "@color-blue-50",
+          "selected": "@color-blue-60"
+        }
+      }
     },
-    "groupedbackground": {
-      "tertiary": {
-        "normal": "@color-gray-90"
-      },
-      "quaternary": {
-        "normal": "@color-gray-80"
+    "mobile-solid_background_default": {
+      "grouped": {
+        "background": {
+          "primary": {
```

Comment:
grouped background primary and secondary are another special case. They vary depending on the theme, which is why they are declared in this file https://github.com/momentum-design/tokens/blob/master/theme-data/dark/dark-bronze.json#:~:text=%22-,groupedbackground,-%22%3A%20%7B